core/src/main/com/github/mmorihiro/core/ArrayActor.kt

```diff
@@ -8,14 +8,18 @@ import com.badlogic.gdx.scenes.scene2d.Actor
 
 internal class ArrayActor : Actor() {
     private val arraySize = 20
-    private val array = Array(arraySize) { BooleanArray(arraySize) { false } }
+    private var array = Array(arraySize) { BooleanArray(arraySize) { false } }
```

Comment:
cf839914a06b1a7455e726547204560952647976 was my mistake. I noticed when implementing 8bc327ef971d75d4c9125b5148b09c2beed38748.
This member should be `var` to make new generation.